src/omg-cli/omg/commands/delete_installation.go

```diff
@@ -58,9 +58,9 @@ func (cmd *DeleteInstallationCommand) run(c *kingpin.ParseContext) error {
 	opsMan := setup.NewService(cfg, envCfg, omSdk, nil, cmd.logger, nil, nil)
 
 	steps := []step{
-		opsMan.PoolTillOnline,
-		opsMan.Unlock,
-		opsMan.DeleteInstallation,
+		{opsMan.PoolTillOnline, "PoolTillOnline"},
```

Comment:
What do you think about initializing the `step` struct with field names? I had to find the struct def to understand what the fields were (which happened to be useful in this case as the struct had changed, but in the future probably wouldn't be)

Reply:
Updated to use field names in initialization. I'm normally a fan of using the field names when initializing structs, and this is no exception